tests/test__collection_api.py

```diff
@@ -5002,6 +4997,30 @@ def test__aggregate_select_nested(self):
             'select_array': [5, 15],
         }, actual[0])
 
+    def test__aggregate_concatArrays(self):
+        self.db.collection.insert_one({
+            'a': [1, 2],
+            'b': ['foo', 'bar', 'baz'],
+            'c': {
+                'arr1': [123],
+            }
+        })
+        actual = self.db.collection.aggregate([{
+            '$project': {
+                'concat': {'$concatArrays': ['$a', ['#', '*'], '$c.arr1', '$b']},
```

Comment:
Please add a test when the parameter of `$concatArrays` is not an array itself `'$concatArrays': 42`.